src/objects/zcl_abapgit_objects_super.clas.abap

```diff
@@ -164,7 +164,6 @@ CLASS ZCL_ABAPGIT_OBJECTS_SUPER IMPLEMENTATION.
         devclass            = iv_package
         master_language     = mv_language
         global_lock         = abap_true
-        author              = sy-uname
```

Comment:
I think the `suppress_dialog`s are safe to add, but I'm not sure about removing `author`

Reply:
Adding the author was a fault by myself done by #2392. If you check all other `RS_CORR_INSERT` calls, the author is never provided.

Reply:
👍